pydy/system.py

```diff
@@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
                 *self._Kane_inlist_insyms()))
         constants.remove(dynamicsymbols._t)
         return constants
+
+    def add_gravity(self, direction=None):
```

Comment:
Point noted! I will create a separate class ``Force`` and ``Gravity`` will be a subclass.